### PR TITLE
Stats command to view statistics

### PR DESCRIPTION
**Pull request by junhui-phoon**
statistics can be view using the following commands:
stats v/all - shows both calorie and weight statistics
stats v/calorie - shows calorie statistics
stats v/weight - show weight statistics

### CODE REVIEW

**Review by ethanso56 — COMMENTED** [review]
looks good to me

File: src/main/java/seedu/address/logic/parser/StatsCommandParser.java [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+        } else if (argMultimap.getValue(PREFIX_VIEW_STATS).get().equals("all")) {
+            showCalorie = true;
+            showWeight = true;
+        } else {
+            throw new ParseException(String.format(MESSAGE_INVALID_COMMAND_FORMAT, StatsCommand.MESSAGE_USAGE));
+        }
+        return new StatsCommand(showCalorie, showWeight);
+    }
+
+    private static boolean arePrefixesPresent(ArgumentMultimap argumentMultimap, Prefix... prefixes) {
```
Comment (ethanso56) [inline src/main/java/seedu/address/logic/parser/StatsCommandParser.java:47]: add javadocs

File: src/main/java/seedu/address/logic/commands/CommandResult.java [hunk 2]
```diff
@@ -17,21 +17,30 @@
...
 
+    /** Statistics should be shown to the user */
+    private final boolean showAllStats;
+    private final boolean showCalorieStats;
+    private final boolean showWeightStats;
+
     /**
      * Constructs a {@code CommandResult} with the specified fields.
      */
-    public CommandResult(String feedbackToUser, boolean showHelp, boolean exit) {
```
Comment (wayne987) [inline src/main/java/seedu/address/logic/commands/CommandResult.java:23]: will it be better if we overload the constructor instead?

Comment (wayne987) [reply src/main/java/seedu/address/logic/commands/CommandResult.java:23]: but other than that the rest looks good

Comment (junhui-phoon) [reply src/main/java/seedu/address/logic/commands/CommandResult.java:23]: actually there is an overloaded constructor for other commands, but i will create one for the stats command for the 3 boolean values.

File: src/main/java/seedu/address/logic/parser/StatsCommandParser.java (hunk 1 above)
Comment (junhui-phoon) [reply src/main/java/seedu/address/logic/parser/StatsCommandParser.java:47]: ok added in the upcoming commit

**Review by wayne987 — APPROVED** [review]
Looks good

**Review by ethanso56 — COMMENTED** [review]
looks good

**Pull request merged**